docker-compose.yml

```diff
@@ -170,9 +170,10 @@ services:
   node:
     image: ghcr.io/danskernesdigitalebibliotek/dpl-go-node:0.0.12
     labels:
-      lagoon.type: node
       provenance: false
-
+      lagoon.type: node-persistent
+      lagoon.persistent: /app/.next
+      lagoon.persistent.size: 500Mi
```

Comment:
Consider adding a comment about why `500Mi` is a reasonable size. 

:+1: for setting a size lower than the default `5Gi` - even though we only pay for what is used. I wonder if there is a reason why you set it at this level. I could not spot anything in the commit comment.